juju/terraform-provider-juju
# [JUJU-4187] Issue & PR templates

## Description

This updates the pull request template and replaces the issue template (that's using old style markdown) with new issue templates separately for bug reports and feature requests using the new Github forms.

Adds a type of change to the PR template for maintenance work.


## QA steps

No complicated QA is needed, maybe wording could be checked. Also in the `Files Changed` section, there's a `View File` button in the top right `...` menu for individual files, which is useful to see what the Gİthub form will look like.
